### ISSUE TITLE
openssl 1.1.0i segmentation fault

### ISSUE THREAD

**Issue by Blackrush** [Homebrew/homebrew-core#31433]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [ ] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link? **=>** `Error: No logs.`
- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

```
tp1 % brew config
HOMEBREW_VERSION: 1.7.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: b119591c6963bff320501f52f9029e4862c00e97
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: c00de10999142dbec42364585e2fc7600dee11a2
Core tap last commit: 10 minutes ago
HOMEBREW_PREFIX: /usr/local
CPU: octa-core 64-bit haswell
Homebrew Ruby: 2.3.7 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/bin/ruby
Clang: 9.1 build 902
Git: 2.18.0 => /usr/local/bin/git
Curl: 7.54.0 => /usr/bin/curl
Java: 1.8.0_131
macOS: 10.13.5-x86_64
CLT: N/A
Xcode: 9.4.1
XQuartz: N/A
tp1 % brew doctor
Your system is ready to brew.
```

To help us debug your issue please explain:
# What you were trying to do (and why)

I am trying to benchmark different cipher algorithms for a university assignment I've got. Here's the command I'm trying to run :

```
tp1 % openssl speed des des-ede3 rc4 rc2 blowfish aes aes-192-cbc aes-256-cbc aes-128-ofb aes-192-ofb aes-256-ofb camellia camellia-192-cbc camellia-256-cbc
```

But here's one that is working : 

```
tp1 % openssl speed des des-ede3 rc4 rc2
```

And two that aren't working : 

```
tp1 % openssl speed camellia-192-cbc
tp1 % openssl speed aes-128-ofb
```

# What happened (include command output)

Straight up segmentation fault :

```
tp1 % openssl speed camellia-192-cbc
[1]    56358 segmentation fault  openssl speed camellia-192-cbc
tp1 % openssl speed aes-128-ofb
[1]    56406 segmentation fault  openssl speed aes-128-ofb
```

# What you expected to happen

Proper benchmarks just like the ones I was getting when running them on 1.1.0h.

# Step-by-step reproduction instructions (by running `brew install` commands)

```
tp1 % brew install openssl@1.1
tp1 % /usr/local/opt/openssl@1.1/bin/openssl speed camellia-192-cbc
```

**Comment by SharpMan** [comment]
Maybe you may try to configure LibreSSL as a back-end, their must be file confusions with the version you use...

**Comment by fxcoudert** [comment]
I'm closing this, as we haven't been able to reproduce this locally, and there was a new version of OpenSSL released since then (1.1.1). Please add new information if the problem persists with this new version, or open a new ticket.
